typespec-tests/src/test/java/com/azure/resourcemanager/resources/ArmResourceTest.java

```diff
@@ -171,7 +171,32 @@ public void testNestedProxyResource() {
         Assertions.assertEquals(RESOURCE_PROVISIONING_STATE, nestedProxyResource.properties().provisioningState());
 
         // NestedProxyResources.delete
-        manager.nesteds()
-            .delete(RESOURCE_GROUP_NAME, TOP_LEVEL_TRACKED_RESOURCE_NAME, NESTED_PROXY_RESOURCE_NAME);
+        manager.nesteds().delete(RESOURCE_GROUP_NAME, TOP_LEVEL_TRACKED_RESOURCE_NAME, NESTED_PROXY_RESOURCE_NAME);
+    }
+
+    @Test
+    public void testSingletonResource() {
```

Comment:
I'm backporting singleton fix to typespec-azure:
https://github.com/Azure/cadl-ranch/pull/759

Reply:
You reminded me that I seems forget to sync test code from core, before this PR.

Make a PR to sync it. https://github.com/Azure/autorest.java/pull/2971